xml2rfc/writers/html.py

```diff
@@ -926,12 +924,11 @@ def render_artwork(self, h, x):
                     if not (svgw and svgh):
                         svgw = float(w)-float(xo)
                         svgh = float(h)-float(yo)
+                elif svgw and svgh:
+                    svg.set('viewBox', '0 0 %s %s' % (svgw, svgh))
```

Comment:
I realize this is a pre-existing concern, but is overwriting any viewBox that might have been specified in the source the right thing to do?

Getting a good understanding of reasonable ways to use viewport and viewbox in the series I think needs some discussion.

Reply:
The accessibility experts recommend the following for understanding viewport and viewBox: https://www.sarasoueidan.com/blog/svg-coordinate-systems/
There's also https://stackoverflow.com/questions/15335926/how-to-use-the-svg-viewbox-attribute